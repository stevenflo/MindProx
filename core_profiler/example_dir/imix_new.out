# FINI: end of program
Dynamic Instruction Count = 99824, Dyn BBL Count = 11282
<DynInsCount> 99824
<BBLCount> 11282
<TotInsCount> 99824
<IntCount> 40054
<IntMulCount> 26
<IntDivCount> 93
<FPCount> 0
<FPMulCount> 0
<FPDivCount> 0
<LdCount> 24815
<StCount> 12651
<Br/CallCount> 19637
<CondBrCount> 17360
<OtherCount> 33352
<SysCallCount> 40

# END_DYNAMIC_STATS