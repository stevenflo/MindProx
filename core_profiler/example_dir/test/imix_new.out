# FINI: end of program
Dynamic Instruction Count = 96980, Dyn BBL Count = 11216
<DynInsCount> 96980
<BBLCount> 11216
<TotInsCount> 96980
<IntCount> 38724
<IntMulCount> 26
<IntDivCount> 93
<FPCount> 0
<FPMulCount> 0
<FPDivCount> 0
<LdCount> 24213
<StCount> 12491
<Br/CallCount> 18903
<CondBrCount> 16752
<OtherCount> 32696
<SysCallCount> 40

# END_DYNAMIC_STATS